### ISSUE TITLE
Support for attribute accessors for Java record type

### ISSUE THREAD

**Issue by jvissers** [npgall/cqengine#303]
Maybe a bit naive, but my initial instinct was to model the values in my lookup set as instances of Java record. However, it seems that attribute generation does not have support for that. Is that something that you want to consider adding for a future version?

**Comment by npgall** [comment]
Yes, that sounds like a good idea. I am planning to dedicate some time to working on CQEngine in the next few weeks, and I will take a look at that!
